apps/meteor/tests/data/livechat/department.ts

```diff
@@ -0,0 +1,61 @@
+import { api, credentials, methodCall, request } from '../api-data';
+import { faker } from '@faker-js/faker';
+import { createUser, login } from '../users.helper';
+import { password } from '../user';
+import { createAgent, makeAgentAvailable } from './rooms';
+import { ILivechatDepartment, IUser } from '@rocket.chat/core-typings';
+
+export const createDepartmentWithMethod = (initialAgents: { agentId: string, username: string }[] = []) =>
+	new Promise((resolve, reject) => {
+		request
+			.post(methodCall('livechat:saveDepartment'))
+			.set(credentials)
+			.send({
+				message: JSON.stringify({
+					method: 'livechat:saveDepartment',
+					params: ['', {
+						enabled: true,
+						email: faker.internet.email(),
+						showOnRegistration: true,
+						showOnOfflineForm: true,
+						name: `new department ${Date.now()}`,
+						description: 'created from api',
+					}, initialAgents],
+					id: 'id',
+					msg: 'method',
+				}),
+			})
+			.end((err: any, res: any) => {
```

Comment:
Hey, i made this file to typescript. Made a "helper" type for responses, that should help you here 😬